content/docs/start/index.md

```diff
@@ -67,6 +67,10 @@ by one in the next few pages:
   data artifacts are built, and provide an efficient way to reproduce them.
   Think "Makefiles for data and ML projects" done right.
 
-- [**Experiments**](/doc/start/experiments) attach parameters, metrics, plots.
-  You can capture and navigate experiments without leaving Git. Think "Git for
-  machine learning".
+- [**ML pipelines**](/doc/start/ml-pipelines) attach parameters, metrics, plots.
+  You can capture, navigate, and evaluate ML projects without leaving Git. Think
+  "Git for machine learning".
```

Comment:
Comparing their descriptions in the index, ML pipelines looks like it'll be a continuation of Data pipelines, as a way to split the old content perhaps? But wasn't this PR about experiments? (Haven't looked at the content yet.)